packages/gateway/src/features/user/schema.graphql

```diff
@@ -150,6 +150,7 @@ input AvatarInput {
 type Query {
   getUser(userId: String): User
   getUserByMobile(mobile: String): User
+  getUserByEmail(emailForNotification: String): User
```

Comment:
yes, in my latest PR i added a type resolver for email so that it always returns the emailForNotification property of user. I wanted to reduce the usage of emailForNotification as params of different method/ property of payloads. So that there is no confusion about these two. User-mgnt and gateway has the usage of emailForNotification from now on.

Reply:
yes, in my latest PR i added a type resolver for email so that it always returns the emailForNotification property of user. I wanted to reduce the usage of emailForNotification as params of different method/ property of payloads. So that there is no confusion about these two. User-mgnt and gateway has the usage of emailForNotification from now on.